doc/source/structures/orbits/orbit.rst

```diff
@@ -197,10 +197,10 @@ Structure
     :type: :ref:`scalar <scalar>`
     :access: Get only
 
-    `mean anomaly`_  in degrees. Even though orbital parameters are
+    `mean anomaly`_  in degrees at orbit epoch. Even though orbital parameters are
     traditionally done in radians, in keeping with the kOS standard
     of making everything into degrees, they are given as degrees by
-    kOS.
+    kOS. Orbit epoch is a complex term, refer to `#1665<https://github.com/KSP-KOS/KOS/issues/1665>`_ for details. 
```

Comment:
There is probably no reason to describe the concept of epoch right now, but let #1665 fix to address that.

This PR is just to save others efforts before #1665 is fixed.